packages/volto/src/components/theme/App/App.jsx

```diff
@@ -51,6 +51,31 @@ import RouteAnnouncer from '@plone/volto/components/theme/RouteAnnouncer/RouteAn
  * @class App
  * @extends {Component}
  */
+//added more languages
+const RTL_LANGUAGES = [
+  'ar',
+  'he',
+  'fa',
+  'ur',
+  'hi',
+  'bn',
+  'pa',
+  'gu',
+  'or',
+  'ta',
+  'te',
+  'kn',
+  'ml',
+  'si',
+  'th',
+  'lo',
+  'my',
+  'km',
+  'vi',
+  'zh',
+  'ja',
+  'ko',
+];
```

Comment:
The list of languages should be reviewed, not all listed here are RTL (for instance ja, see @terapyon's comment). As a reference this is the list of RTL languages in Classic: https://github.com/plone/plone.app.layout/blob/master/plone/app/layout/globals/portal.py#L21